app/models/ncr/work_order.rb

```diff
@@ -26,6 +26,7 @@ def self.purchase_amount_column_name
     before_validation :normalize_values
     before_update :record_changes
 
+    validates :amount, presence: true
```

Comment:
I think not having it for this is fine since we don't have any other simple validation specs. 

I know a lot of people don't like simple validation specs because it's kind of like testing Rails, but I do like it them because using them makes specs a full record of documentation for the code they cover. It's nice being able to look into a spec file and know everything you need to know about an object. 

BUT that is something for next time - for now, seems like a validation isn't needed.